README.rst

```diff
@@ -450,6 +450,69 @@ The following options (``[flags]``) are supported:
   Defaults to ``./instances.yml`` (or to the value of the "cfg"
   parameter in the Cartridge `configuration file <Overriding default options_>`_).
 
+  The ``instances.yml`` file contains parameters for starting Cartridge
+  application instances and is placed in the application root directory.
+  These parameters are then passed to the `cartridge.cfg <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_api/modules/cartridge/#cfg-opts-box-opts>`_
+  function.
+
+  Example of the ``instances.yml`` file:
+
+  .. code-block:: yaml
+
+      myapp.router:
+        advertise_uri: localhost:3301
+        http_port: 8081
+
+      myapp.s1-master:
+        advertise_uri: localhost:3302
+        http_port: 8082
+
+  Parameters that can be specified in ``instances.yml`` are of two kinds:
+
+  * `cartridge.cfg <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_api/modules/cartridge.argparse/#tables>`_
+    parameters
+  * custom parameters.
+
+  Among the ``cartridge.cfg`` parameters, ``advertise_uri`` is required while
+  others are optional.
+
+  .. note::
+
+     Some of the ``cartridge.cfg`` parameters are exceptions: they passed by
+     Catridge CLI itself and cannot be specified in the ``instances.yml`` file:
+
+     * ``workdir``
+     * ``console_sock``
+     * ``pid_file``.
```

Comment:
They're not exceptions. They're passed too, but CLI overrides them with environment variables of his own.

Reply:
Yes, but on `cartridge start`  you have no way to overwrite these values.

Reply:
re-written